<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 10 Differential Abundance | Complete Microbiome Analysis</title>
  <meta name="description" content="A work-in-progress" />
  <meta name="generator" content="bookdown 0.27 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 10 Differential Abundance | Complete Microbiome Analysis" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="A work-in-progress" />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 10 Differential Abundance | Complete Microbiome Analysis" />
  
  <meta name="twitter:description" content="A work-in-progress" />
  

<meta name="author" content="Alana Schick" />


<meta name="date" content="2022-06-17" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="beta-diversity.html"/>
<link rel="next" href="normalization-methods.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Complete Microbiome Analysis</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#overview-of-sections"><i class="fa fa-check"></i>Overview of sections</a></li>
</ul></li>
<li class="part"><span><b>I Part 1: Samples to Sequences</b></span></li>
<li class="chapter" data-level="1" data-path="experimental-design-considerations.html"><a href="experimental-design-considerations.html"><i class="fa fa-check"></i><b>1</b> Experimental Design Considerations</a>
<ul>
<li class="chapter" data-level="1.1" data-path="experimental-design-considerations.html"><a href="experimental-design-considerations.html#amplicon-versus-shotgun"><i class="fa fa-check"></i><b>1.1</b> Amplicon versus Shotgun</a></li>
<li class="chapter" data-level="1.2" data-path="experimental-design-considerations.html"><a href="experimental-design-considerations.html#s-resources"><i class="fa fa-check"></i><b>1.2</b> 16S Resources</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="power-analyses.html"><a href="power-analyses.html"><i class="fa fa-check"></i><b>2</b> Power Analyses</a></li>
<li class="chapter" data-level="3" data-path="sequencing-samples.html"><a href="sequencing-samples.html"><i class="fa fa-check"></i><b>3</b> Sequencing samples</a></li>
<li class="part"><span><b>II Working with Sequence Data</b></span></li>
<li class="chapter" data-level="4" data-path="filtering-and-cleaning.html"><a href="filtering-and-cleaning.html"><i class="fa fa-check"></i><b>4</b> Filtering and cleaning</a></li>
<li class="chapter" data-level="5" data-path="generating-abundance-tables.html"><a href="generating-abundance-tables.html"><i class="fa fa-check"></i><b>5</b> Generating Abundance Tables</a>
<ul>
<li class="chapter" data-level="5.1" data-path="generating-abundance-tables.html"><a href="generating-abundance-tables.html#dada2-notes"><i class="fa fa-check"></i><b>5.1</b> Dada2 notes</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="remove-contaminants.html"><a href="remove-contaminants.html"><i class="fa fa-check"></i><b>6</b> Remove Contaminants</a></li>
<li class="chapter" data-level="7" data-path="functional-annotation.html"><a href="functional-annotation.html"><i class="fa fa-check"></i><b>7</b> Functional Annotation</a>
<ul>
<li class="chapter" data-level="7.1" data-path="functional-annotation.html"><a href="functional-annotation.html#amplicon-sequences"><i class="fa fa-check"></i><b>7.1</b> Amplicon sequences</a>
<ul>
<li class="chapter" data-level="7.1.1" data-path="functional-annotation.html"><a href="functional-annotation.html#picrust2-notes"><i class="fa fa-check"></i><b>7.1.1</b> Picrust2 notes</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="functional-annotation.html"><a href="functional-annotation.html#shotgun-sequences"><i class="fa fa-check"></i><b>7.2</b> Shotgun sequences</a></li>
</ul></li>
<li class="part"><span><b>III Working with Abundance Tables</b></span></li>
<li class="chapter" data-level="8" data-path="alpha-diversity.html"><a href="alpha-diversity.html"><i class="fa fa-check"></i><b>8</b> Alpha diversity</a></li>
<li class="chapter" data-level="9" data-path="beta-diversity.html"><a href="beta-diversity.html"><i class="fa fa-check"></i><b>9</b> Beta Diversity</a></li>
<li class="chapter" data-level="10" data-path="differential-abundance.html"><a href="differential-abundance.html"><i class="fa fa-check"></i><b>10</b> Differential Abundance</a>
<ul>
<li class="chapter" data-level="10.1" data-path="differential-abundance.html"><a href="differential-abundance.html#corncob-notes"><i class="fa fa-check"></i><b>10.1</b> Corncob notes</a></li>
<li class="chapter" data-level="10.2" data-path="differential-abundance.html"><a href="differential-abundance.html#deseq2-notes"><i class="fa fa-check"></i><b>10.2</b> DESeq2 notes</a></li>
<li class="chapter" data-level="10.3" data-path="differential-abundance.html"><a href="differential-abundance.html#metagenomeseq"><i class="fa fa-check"></i><b>10.3</b> metagenomeSeq</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="normalization-methods.html"><a href="normalization-methods.html"><i class="fa fa-check"></i><b>11</b> Normalization Methods</a></li>
<li class="chapter" data-level="12" data-path="multivariate-analysis.html"><a href="multivariate-analysis.html"><i class="fa fa-check"></i><b>12</b> Multivariate Analysis</a></li>
<li class="part"><span><b>IV Advanced Topics</b></span></li>
<li class="chapter" data-level="13" data-path="reproducible-workflows.html"><a href="reproducible-workflows.html"><i class="fa fa-check"></i><b>13</b> Reproducible Workflows</a>
<ul>
<li class="chapter" data-level="13.1" data-path="reproducible-workflows.html"><a href="reproducible-workflows.html#snakemake"><i class="fa fa-check"></i><b>13.1</b> Snakemake</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="multiomic-methods.html"><a href="multiomic-methods.html"><i class="fa fa-check"></i><b>14</b> Multiomic Methods</a>
<ul>
<li class="chapter" data-level="14.1" data-path="multiomic-methods.html"><a href="multiomic-methods.html#correlation-analysis"><i class="fa fa-check"></i><b>14.1</b> Correlation Analysis</a></li>
<li class="chapter" data-level="14.2" data-path="multiomic-methods.html"><a href="multiomic-methods.html#network-analysis"><i class="fa fa-check"></i><b>14.2</b> Network Analysis</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Complete Microbiome Analysis</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="differential-abundance" class="section level1 hasAnchor" number="10">
<h1><span class="header-section-number">Chapter 10</span> Differential Abundance<a href="differential-abundance.html#differential-abundance" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Comparing methods:</p>
<p><a href="https://www.nicholas-ollberding.com/post/identifying-differentially-abundant-features-in-microbiome-data/" class="uri">https://www.nicholas-ollberding.com/post/identifying-differentially-abundant-features-in-microbiome-data/</a></p>
<div id="corncob-notes" class="section level2 hasAnchor" number="10.1">
<h2><span class="header-section-number">10.1</span> Corncob notes<a href="differential-abundance.html#corncob-notes" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Beta binomial regression.</p>
<p>How to interpret coefficients:</p>
<p>By default, corncob uses the logit link between the expected relative abundance and the covariates. This can be interpreted on the log-odds scale, similar to with logistic regression. Typically, I recommend users interpret parameters directly on the logit scale, for example “The expected difference in the logit-transformed relative abundance between two samples that differ by one unit change in [my covariate] controlling for [all controls] is [coefficient value]”. If you want to interpret more directly, I’d recommend looking up the interpretation of log-odds, or logistic regression parameters. Unfortunately, the definition is not intuitive, and it is necessary to use a link function in order to appropriately model something on the [0,1] scale, such as relative abundance.</p>
<p><a href="https://github.com/bryandmartin/corncob/issues/68" class="uri">https://github.com/bryandmartin/corncob/issues/68</a>
(Note: I have ideas (but not time) about how to improve this feature)</p>
<p><a href="https://github.com/bryandmartin/corncob/blob/master/R/print_differentialTest.R" class="uri">https://github.com/bryandmartin/corncob/blob/master/R/print_differentialTest.R</a>
(This is the code for the function that performs this differential test)</p>
</div>
<div id="deseq2-notes" class="section level2 hasAnchor" number="10.2">
<h2><span class="header-section-number">10.2</span> DESeq2 notes<a href="differential-abundance.html#deseq2-notes" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>DESeq2: normalizes by estimating the negative binomial distribution for each taxon in each sample
<a href="https://bioconductor.org/packages/devel/bioc/vignettes/phyloseq/inst/doc/phyloseq-mixture-models.html" class="uri">https://bioconductor.org/packages/devel/bioc/vignettes/phyloseq/inst/doc/phyloseq-mixture-models.html</a></p>
<p>DESEQ analysis on RNA-seq data:
<a href="https://compbiocore.github.io/deseq-workshop-1/assets/deseq_workshop_1.html" class="uri">https://compbiocore.github.io/deseq-workshop-1/assets/deseq_workshop_1.html</a></p>
<p>Analyzing RNA-seq data with DESeq2:
<a href="http://bioconductor.org/packages/devel/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#contrasts" class="uri">http://bioconductor.org/packages/devel/bioc/vignettes/DESeq2/inst/doc/DESeq2.html#contrasts</a></p>
</div>
<div id="metagenomeseq" class="section level2 hasAnchor" number="10.3">
<h2><span class="header-section-number">10.3</span> metagenomeSeq<a href="differential-abundance.html#metagenomeseq" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>MetagenomeSeq: uses sample quantiles to normalize accounting for undersampling.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="beta-diversity.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="normalization-methods.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/differential-abundance.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["complete-microbiome-analysis.pdf", "complete-microbiome-analysis.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
